### PR TITLE
[RDAPI-28103] remove from aug 22 release notes

### PR DESCRIPTION
**Pull request by fprendergast**
http://pr-2477.axway-open-docs.opendocs-builder.pcloud.axway.int/docs/apim_administration/apimgr_admin/api_mgmt_virtualize_web/#virtualize-a-rest-api-as-a-front-end-api

http://pr-2477.axway-open-docs.opendocs-builder.pcloud.axway.int/docs/apim_policydev/apigw_gw_instances/general_virtual_hosts/#configure-virtual-hosts-for-http-services

http://pr-2477.axway-open-docs.opendocs-builder.pcloud.axway.int/docs/apim_policydev/apigw_web_services/general_policy_wsdl/#import-a-wsdl-file

http://pr-2477.axway-open-docs.opendocs-builder.pcloud.axway.int/docs/apim_administration/apimgr_admin/api_mgmt_virtualize_web/

### CODE REVIEW

**Comment by docsbuilder** [comment]
### <span aria-hidden="true">✅</span> Deploy preview is ready!

 |  Name | Link |
 |---------------------------------|------------------------|
 |<span aria-hidden="true">🔨</span> Latest commit | bc0d88a |
 |<span aria-hidden="true">😎</span> Deploy preview | http://PR-2477.axway-open-docs.opendocs-builder.pcloud.axway.int |

File: content/en/docs/apim_administration/apimgr_admin/api_mgmt_virtualize_web.md [hunk 1]
```diff
@@ -40,6 +40,34 @@ When you have first registered a back-end REST API in API Manager, you can then
 
 {{< alert title="Note" color="primary" >}}If the virtualized API is not a Swagger 2.0-compatible REST API, API Manager displays a message that Swagger download will not be available in the **API Catalog**.{{< /alert >}}
 
+{{< alert title="Note" color="primary" >}}When exporting Swagger 2.0 from API Manager Catalog with multiple traffic ports defined, if you try to subsequently reimport the generated document into another API Manager, the back-end URLs (HTTP and HTTPS) are correctly generated but the port will be incorrect for one of the URLs.
```
Comment (lcattolico) [inline content/en/docs/apim_administration/apimgr_admin/api_mgmt_virtualize_web.md:43]: Would it be better to have only one Note section?

**Review by lcattolico — APPROVED** [review]

File: content/en/docs/apim_administration/apimgr_admin/api_mgmt_virtualize_web.md (hunk 1 above)
Comment (fprendergast) [reply content/en/docs/apim_administration/apimgr_admin/api_mgmt_virtualize_web.md]: Yes, as much as it pains me you are probably right.

**Comment by lcattolico** [comment]
Much better now, thanks @fprendergast

File: content/en/docs/apim_relnotes/20220830_apimgr_relnotes.md [hunk 2]
```diff
@@ -214,52 +214,6 @@ def invoke(msg):
 
 Related Issue: RDAPI-21363
 
-### API Gateway web service WSDL schema validation failure
```
Comment (andreamussap) [inline content/en/docs/apim_relnotes/20220830_apimgr_relnotes.md:217]: @fprendergast I believe this update should be done in the November 22 release notes, not in Aug 22

File: content/en/docs/apim_administration/apimgr_admin/api_mgmt_virtualize_web.md [hunk 3]
```diff
@@ -38,7 +38,35 @@ When you have first registered a back-end REST API in API Manager, you can then
 
 ![Imported API details in the web console](/Images/docbook/images/api_mgmt/api_mgmt_frontend_api_edit.png)
 
-{{< alert title="Note" color="primary" >}}If the virtualized API is not a Swagger 2.0-compatible REST API, API Manager displays a message that Swagger download will not be available in the **API Catalog**.{{< /alert >}}
+{{< alert title="Note" color="primary" >}}If the virtualized API is not a Swagger 2.0-compatible REST API, API Manager displays a message that Swagger download will not be available in the **API Catalog**.
```
Comment (andreamussap) [inline content/en/docs/apim_administration/apimgr_admin/api_mgmt_virtualize_web.md:41]: This is too long to be a "Note", it lost its purpose. Please create a subsection (heading 3) for this information, e.g.:

- "Virtualized APIs must be Swagger 2.0-compatible to be available in the API Catalog" ??
- "API Catalog displays Swagger 2.0-compatible APIs only" ??
- "Download from API Catalog is only available for Swagger 2.0-compatible APIs" ??

File: content/en/docs/apim_policydev/apigw_gw_instances/general_virtual_hosts.md [hunk 4]
```diff
@@ -46,6 +46,10 @@ You can configure virtual hosts at the HTTP service level. This means that these
    * **Enabled**: Select whether the virtual host processing is enabled. This is enabled by default.
    * **Hosts**: Specify the list of domains that you wish to host under this HTTP service. To add a host, click **Add** at the bottom right, and enter the domain name (for example `company1.api.example.com`). You can also specify domain names using wildcards already configured in your DNS (for example `*.example.com:/8080` or `company3.api.example.com.*`).
 
+{{< alert title="Note" color="primary" >}}In API Manager, if a virtual host (global default, organization level, or for a published API) is set to `myhost` and there are multiple traffic ports (mix of HTTP and HTTPS) configured, API Manager correctly displays `https://myhost` and `http://myhost` as base path URLs in the API Catalog.
```
Comment (andreamussap) [inline content/en/docs/apim_policydev/apigw_gw_instances/general_virtual_hosts.md:49]: There's no reason for this to be a Note. It's just more information about the section. Could you please remove the Note format, and just leave the text as a paragraph, or if you want to call out for this information, create a heading 3 section, with a relevant title before it

**Comment by andreamussap** [comment]
> Much better now, thanks @fprendergast

@lcattolico  Wouldn't be interesting to add a new note to the release notes to say that you've removed those know issues and they're not described in the latest docs?

**Comment by lcattolico** [comment]
@andreamussap I don't think we usually would add a new note to the release notes to say that we have moved known issues, we already highlighted those problems in the previous release, in my opinion is not necessary.

File: content/en/docs/apim_policydev/apigw_gw_instances/general_virtual_hosts.md (hunk 4 above)
Comment (fprendergast) [reply content/en/docs/apim_policydev/apigw_gw_instances/general_virtual_hosts.md]: Removed note and added a paragraph in it's place with new heading.

**Pull request merged**